app/main/index.ts

```diff
@@ -362,6 +364,18 @@ app.on('ready', () => {
 	ipcMain.on('save-last-tab', (_event: Electron.IpcMessageEvent, index: number) => {
 		ConfigUtil.setConfigItem('lastActiveTab', index);
 	});
+
+	ipcMain.on('leveldb-set-item', (_event: Electron.IpcMessageEvent, key: string, value: any) => {
```

Comment:
Can we name this only db-set-item if possible? Thats sounds better. :)

Reply:
Sure, thanks for the suggestion!

Reply:
Made the changes. Resolving this conversation.